From ab05bdd4b68e33270c78b7316f83bbdf93c36310 Mon Sep 17 00:00:00 2001
From: Jenkins Slave <jenkins@code.rdkcentral.com>
Date: Thu, 7 Sep 2023 11:21:27 +0100
Subject: [PATCH] added-source-code-changes-for-systemservice-for-fwup

---
 SystemServices/SystemServices.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SystemServices/SystemServices.cpp b/SystemServices/SystemServices.cpp
index 7ae77e36..b854882f 100644
--- a/SystemServices/SystemServices.cpp
+++ b/SystemServices/SystemServices.cpp
@@ -1235,7 +1235,7 @@ namespace WPEFramework {
                 JsonObject& response)
         {
             LOGWARN("SystemService updatingFirmware\n");
-            string command("/lib/rdk/swupdate_utility.sh 0 4 >> /opt/logs/swupdate.log &");
+            string command("/lib/rdk/swupdate_utility.sh 3 1 >> /opt/logs/swupdate.log &");
             Utils::cRunScript(command.c_str());
             returnResponse(true);
         }
-- 
2.37.3

